app/src/main/res/values/strings.xml

```diff
@@ -292,11 +293,13 @@
   <string name="app_language">App Language</string>
   <string name="story_text_size">Story Text Size</string>
   <string name="profile">Profile</string>
+  <string name="story_count_text_view">2 Stories</string>
+
   <string name="topics_in_progress">Topics in Progress</string>
   <string name="topic_in_progress">Topic in Progress</string>
   <string name="stories_completed">Stories Completed</string>
   <string name="story_completed">Story Completed</string>
   <!-- WalkthroughActivity -->
   <string name="walkthrough_welcome_description">Learn new math skills with stories that show you how to use them in your daily life</string>
   <string name="welcome">"Welcome %s!"</string>
-</resources>
+</resources>
```

Comment:
EOF is missing, add one blank line at the end.

Reply:
Added